packages/cli/src/oclif/commands/versions.js

```diff
@@ -22,6 +22,12 @@ class VersionCommand extends BaseCommand {
       emptyMessage:
         'No versions to show. Try adding one with the `zapier push` command',
     });
+
+    if (versions[0].user_count === null) {
+      this.warn(
+        'User counts are still being calculated - run this command again in ~10 seconds (or longer if your integration has lots of users).'
```

Comment:
The user count for each version is calculated independently, so it's possible that some versions have the count while some others don't.

So instead of only checking the first version, we should check all the versions. If there's any version having a null `user_count`. We should show this warning.

Also, I'd probably say "**Some** user counts are still being calculated" to avoid any confusion.

Reply:
Ah, fair point! Got lucky on an app having an exact test case:

![](https://cdn.zappy.app/14634a2c6189a5b4eb76a127db9582ec.png)

Fixed in 9940fa68